gdnative-core/src/class.rs

```diff
@@ -285,6 +285,16 @@ impl<T: NativeClass> Instance<T> {
     }
 }
 
+impl<T> Default for Instance<T>
+where
+    T: NativeClass,
+    T::Base: Instanciable,
+{
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
`Default` shouldn't be implemented for `Instance`. `Instance::new` performs an expensive operation (creating the object), while `Default` has the connotation of being virtually free. This one should be silenced instead.